### ISSUE TITLE
Assginment 2

### ISSUE THREAD

**Issue by barodeur** [ashlynbaum/yogainthepark#4]
I want the API to be documented in a `API.md` file before starting the implementation of it.

The way you format that file is up to you but I should be able to easily find the **endpoints** and read about the parameters.

Here are some examples of well documented API :
- https://developer.github.com/v3/
- https://instagram.com/developer/endpoints/

**Comment by barodeur** [comment]
Because I ask you to write a `.md` file, you'll need a markdown editor. Mardown is just text, you can edit it in `vim`, `nano`, sublime or any text editor, but it's nice to have a live rendering of your mardown. You can use [Mou.app](http://25.io/mou/).

feel free to correct my english
